spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -289,75 +314,102 @@ def call_with_params(params)
 
       end
     end
+  end
+
+  describe "update" do
+    let!(:update_user) { create(:user) }
```

Comment:
Thanks, I think I originally had trouble with the variable being unbound or apparently bound to a different user, but then never changed it once the problem was resolved!